lib/OrganizationsFieldArray/OrganizationCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AppIcon } from '@folio/stripes/core';
+import { FormattedMessage } from 'react-intl';
+import { get, isEmpty } from 'lodash';
+import {
+  Card,
+  Icon,
+  MultiColumnList,
+  Layout,
+} from '@folio/stripes/components';
+import css from './OrganizationCard.css';
+
+export default class OrganizationCard extends React.Component {
+  static propTypes = {
+    cardStyle: PropTypes.string,
```

Comment:
I get why you're passing `cardStyle` in here but the whole point of this card is so that we don't have to remember what particular props to set on `Card` to have the right styling. At the very least, this should probably have a default prop value so that common usage of rendering an organization doesn't require `cardStyle` to be passed in.

It's part of what makes me unsure whether it's actually appropriate for this component to be used when rendering both a selected and unselected organization. I kind of feel like this should only be used for a _selected_ organization. We can talk about this if you want.